### PR TITLE
CheckBox: temporarily disable inkRipple untill it fixed

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: js/renovation/ui/__tests__/check_box.test.tsx [hunk 1]
```diff
@@ -638,10 +638,10 @@ describe('CheckBox', () => {
       afterEach(() => jest.resetAllMocks());
 
       describe('useInkRiple', () => {
-        it('should be true if material theme', () => {
+        it('should be true (temporaly replaced with false) if material theme', () => {
```
Comment (ksercs) [inline js/renovation/ui/__tests__/check_box.test.tsx]: we should just skip this test with no any other changes and add comment-note at the test begining

File: js/renovation/ui/check_box.tsx [hunk 2]
```diff
@@ -137,9 +137,12 @@ export const defaultOptionRules = createDefaultOptionRules<CheckBoxProps>([{
   device: (): boolean => devices.real().deviceType === 'desktop' && !devices.isSimulator(),
   options: { focusStateEnabled: true },
 }, {
+  // NOTE: useInkRipple temporarily set to false
+  //       untill inlRipple styles dosn't
+  //       look properly.
```
Comment (ksercs) [inline js/renovation/ui/check_box.tsx]: let's make it more short: `it's disabled until styles fix: see *card link*`

**Review by ksercs — DISMISSED** [review]

File: js/renovation/ui/__tests__/check_box.test.tsx [hunk 3]
```diff
@@ -638,7 +638,9 @@ describe('CheckBox', () => {
       afterEach(() => jest.resetAllMocks());
 
       describe('useInkRiple', () => {
-        it('should be true if material theme', () => {
+        // NOTE: it's disabled until styles fix: see https://trello.com/c/5Pbm18YA/4165-enable-inkripple-after-fixing-styles
```
Comment (ksercs) [inline js/renovation/ui/__tests__/check_box.test.tsx]:
```suggestion
```

File: js/renovation/ui/check_box.tsx [hunk 4]
```diff
@@ -137,9 +137,10 @@ export const defaultOptionRules = createDefaultOptionRules<CheckBoxProps>([{
   device: (): boolean => devices.real().deviceType === 'desktop' && !devices.isSimulator(),
   options: { focusStateEnabled: true },
 }, {
+  // NOTE: it's disabled until styles fix: see https://trello.com/c/5Pbm18YA/4165-enable-inkripple-after-fixing-styles
```
Comment (ksercs) [inline js/renovation/ui/check_box.tsx]:
```suggestion
  // NOTE: it's disabled until styles fix: see https://trello.com/c/5Pbm18YA
```

**Review by ksercs — DISMISSED** [review]

**Review by ksercs — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**